actions/websocket_actions.jsx

```diff
@@ -248,7 +248,7 @@ function handleClose(failCount) {
     ]));
 }
 
-function handleEvent(msg) {
+export function handleEvent(msg) {
```

Comment:
@hmhealey 
for testing i see we've exported other functions, i think it'd be better to just export this and test this rather than testing individual functions. Will need to un-export those other functions to keep things tidy. Do you mind un-exporting functions that you updated and test via `handleEvent` instead?

Reply:
I can understand why you'd want to keep the exported API clean, but I prefer to test the inner function whenever possible to keep the scope of the test narrow. Unfortunately, JS doesn't have any good options for that without breaking up this file, so I'd rather sacrifice the API cleanliness to get better test coverage

Reply:
Agreed. I think that's a good indicator that we _should_ split up the file horizontally. In this case I would propose creating a new file called `websocket_handlers.js` that exposes all the websocket action handlers. Doesn't have to happen as part of this change but i think that's the direction we should move towards. Want me to create a jira for it?

Reply:
Yeah, that'd be a good idea as its own ticket. I'd even say that we could move it into its own folder, so we could have something like `actions/websocket/index.js` and `actions/websocket/handlers.js`

Reply:
make sense 👍 Created a jira and assigned it to the apps team. A good HW candidate for someone interesting in QA?

https://mattermost.atlassian.net/browse/MM-17894